admin-dev/themes/default/js/bundle/product/product-combinations.js

```diff
@@ -217,6 +217,13 @@ var combinations = (function() {
           number.text(countSelectedProducts() + '/' + allProductCombination);
         });
 
+        $(function() {
```

Comment:
For future reference, please add a comment stating what this piece of code is supposed to do

For example: "Add title on product's combination image"